Warnings since a592b0605a431e5e121cd747471d2e769d8c95fe

I have a lot of warnings filling up my Logs tab since a592b0605a431e5e121cd747471d2e769d8c95fe
Is that expected @SurfaceS ?

It's a lot of this:

```
17:00:14 WARN  Could not hydrate device or its services from descriptor: (RemoteDevice) Identity: (RemoteDeviceIdentity) UDN: uuid:bc51db50-7984-44af-947b-a90f538ed0dd, Descriptor: http://192.168.1.7:1946/, Root: true
org.jupnp.binding.xml.DescriptorBindingException: Could not parse device descriptor
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:115)
	at org.jupnp.binding.xml.RecoveringUDA10DeviceDescriptorBinderImpl.describe(RecoveringUDA10DeviceDescriptorBinderImpl.java:54)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:187)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:175)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.run(RetrieveRemoteDescriptors.java:106)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.jupnp.binding.xml.DescriptorBindingException: Could not parse device DOM
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:136)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:110)
	... 7 common frames omitted
Caused by: org.jupnp.model.types.InvalidValueException: Can't parse device type string (namespace/type/version): urn:dial-multiscreen-org:service:dial:1
	at org.jupnp.model.types.DeviceType.valueOf(DeviceType.java:128)
	at org.jupnp.binding.staging.MutableDevice.createDeviceType(MutableDevice.java:82)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:73)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:64)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.buildInstance(UDA10DeviceDescriptorBinderImpl.java:142)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:131)
	... 8 common frames omitted
17:00:14 WARN  Regular parsing failed
org.jupnp.binding.xml.DescriptorBindingException: Could not parse device descriptor
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:115)
	at org.jupnp.binding.xml.RecoveringUDA10DeviceDescriptorBinderImpl.describe(RecoveringUDA10DeviceDescriptorBinderImpl.java:54)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:187)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:175)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.run(RetrieveRemoteDescriptors.java:106)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.jupnp.binding.xml.DescriptorBindingException: Could not parse device DOM
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:136)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:110)
	... 7 common frames omitted
Caused by: org.jupnp.model.types.InvalidValueException: Can't parse device type string (namespace/type/version): urn:dial-multiscreen-org:service:dial:1
	at org.jupnp.model.types.DeviceType.valueOf(DeviceType.java:128)
	at org.jupnp.binding.staging.MutableDevice.createDeviceType(MutableDevice.java:82)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:73)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:64)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.buildInstance(UDA10DeviceDescriptorBinderImpl.java:142)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:131)
	... 8 common frames omitted
17:00:14 WARN  Removing leading garbage didn't work
org.jupnp.binding.xml.DescriptorBindingException: Could not parse device descriptor
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:115)
	at org.jupnp.binding.xml.RecoveringUDA10DeviceDescriptorBinderImpl.describe(RecoveringUDA10DeviceDescriptorBinderImpl.java:78)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:187)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:175)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.run(RetrieveRemoteDescriptors.java:106)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.jupnp.binding.xml.DescriptorBindingException: Could not parse device DOM
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:136)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:110)
	... 7 common frames omitted
Caused by: org.jupnp.model.types.InvalidValueException: Can't parse device type string (namespace/type/version): urn:dial-multiscreen-org:service:dial:1
	at org.jupnp.model.types.DeviceType.valueOf(DeviceType.java:128)
	at org.jupnp.binding.staging.MutableDevice.createDeviceType(MutableDevice.java:82)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:73)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:64)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.buildInstance(UDA10DeviceDescriptorBinderImpl.java:142)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:131)
	... 8 common frames omitted
17:00:14 WARN  Could not hydrate device or its services from descriptor: (RemoteDevice) Identity: (RemoteDeviceIdentity) UDN: uuid:bc51db50-7984-44af-947b-a90f538ed0dd, Descriptor: http://192.168.1.7:1946/, Root: true
org.jupnp.binding.xml.DescriptorBindingException: Could not parse device descriptor
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:115)
	at org.jupnp.binding.xml.RecoveringUDA10DeviceDescriptorBinderImpl.describe(RecoveringUDA10DeviceDescriptorBinderImpl.java:54)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:187)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:175)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.run(RetrieveRemoteDescriptors.java:106)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.jupnp.binding.xml.DescriptorBindingException: Could not parse device DOM
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:136)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:110)
	... 7 common frames omitted
Caused by: org.jupnp.model.types.InvalidValueException: Can't parse device type string (namespace/type/version): urn:dial-multiscreen-org:service:dial:1
	at org.jupnp.model.types.DeviceType.valueOf(DeviceType.java:128)
	at org.jupnp.binding.staging.MutableDevice.createDeviceType(MutableDevice.java:82)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:73)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:64)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.buildInstance(UDA10DeviceDescriptorBinderImpl.java:142)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:131)
	... 8 common frames omitted
17:00:14 WARN  Regular parsing failed
org.jupnp.binding.xml.DescriptorBindingException: Could not parse device descriptor
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:115)
	at org.jupnp.binding.xml.RecoveringUDA10DeviceDescriptorBinderImpl.describe(RecoveringUDA10DeviceDescriptorBinderImpl.java:54)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:187)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:175)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.run(RetrieveRemoteDescriptors.java:106)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.jupnp.binding.xml.DescriptorBindingException: Could not parse device DOM
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:136)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:110)
	... 7 common frames omitted
Caused by: org.jupnp.model.types.InvalidValueException: Can't parse device type string (namespace/type/version): urn:dial-multiscreen-org:service:dial:1
	at org.jupnp.model.types.DeviceType.valueOf(DeviceType.java:128)
	at org.jupnp.binding.staging.MutableDevice.createDeviceType(MutableDevice.java:82)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:73)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:64)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.buildInstance(UDA10DeviceDescriptorBinderImpl.java:142)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:131)
	... 8 common frames omitted
17:00:14 WARN  Removing leading garbage didn't work
org.jupnp.binding.xml.DescriptorBindingException: Could not parse device descriptor
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:115)
	at org.jupnp.binding.xml.RecoveringUDA10DeviceDescriptorBinderImpl.describe(RecoveringUDA10DeviceDescriptorBinderImpl.java:78)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:187)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:175)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.run(RetrieveRemoteDescriptors.java:106)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.jupnp.binding.xml.DescriptorBindingException: Could not parse device DOM
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:136)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:110)
	... 7 common frames omitted
Caused by: org.jupnp.model.types.InvalidValueException: Can't parse device type string (namespace/type/version): urn:dial-multiscreen-org:service:dial:1
	at org.jupnp.model.types.DeviceType.valueOf(DeviceType.java:128)
	at org.jupnp.binding.staging.MutableDevice.createDeviceType(MutableDevice.java:82)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:73)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:64)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.buildInstance(UDA10DeviceDescriptorBinderImpl.java:142)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:131)
	... 8 common frames omitted
17:00:14 WARN  Could not hydrate device or its services from descriptor: (RemoteDevice) Identity: (RemoteDeviceIdentity) UDN: uuid:bc51db50-7984-44af-947b-a90f538ed0dd, Descriptor: http://192.168.1.7:1946/, Root: true
org.jupnp.binding.xml.DescriptorBindingException: Could not parse device descriptor
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:115)
	at org.jupnp.binding.xml.RecoveringUDA10DeviceDescriptorBinderImpl.describe(RecoveringUDA10DeviceDescriptorBinderImpl.java:54)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:187)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.describe(RetrieveRemoteDescriptors.java:175)
	at org.jupnp.protocol.RetrieveRemoteDescriptors.run(RetrieveRemoteDescriptors.java:106)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.jupnp.binding.xml.DescriptorBindingException: Could not parse device DOM
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:136)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:110)
	... 7 common frames omitted
Caused by: org.jupnp.model.types.InvalidValueException: Can't parse device type string (namespace/type/version): urn:dial-multiscreen-org:service:dial:1
	at org.jupnp.model.types.DeviceType.valueOf(DeviceType.java:128)
	at org.jupnp.binding.staging.MutableDevice.createDeviceType(MutableDevice.java:82)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:73)
	at org.jupnp.binding.staging.MutableDevice.build(MutableDevice.java:64)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.buildInstance(UDA10DeviceDescriptorBinderImpl.java:142)
	at org.jupnp.binding.xml.UDA10DeviceDescriptorBinderImpl.describe(UDA10DeviceDescriptorBinderImpl.java:131)
	... 8 common frames omitted
```

--- a/src/main/java/net/pms/network/mediaserver/jupnp/UmsUpnpServiceConfiguration.java
+++ b/src/main/java/net/pms/network/mediaserver/jupnp/UmsUpnpServiceConfiguration.java
@@ -477,16 +477,16 @@ public class UmsUpnpServiceConfiguration implements UpnpServiceConfiguration {
 	}
 
 	private static void resetLoggingMode() {
-		Level mediaServerLevel = CONFIGURATION.isUpnpDebugMediaServer() ? Level.ALL : Level.INFO;
-		Level basicLevel = CONFIGURATION.isUpnpDebugBasic() ? Level.ALL : Level.INFO;
-		Level fullLevel = CONFIGURATION.isUpnpDebugFull() ? Level.ALL : Level.INFO;
-		if (fullLevel == Level.ALL) {
+		Level mediaServerLevel = CONFIGURATION.isUpnpDebugMediaServer() ? Level.TRACE : Level.INFO;
+		Level basicLevel = CONFIGURATION.isUpnpDebugBasic() ? Level.TRACE : Level.INFO;
+		Level fullLevel = CONFIGURATION.isUpnpDebugFull() ? Level.TRACE : Level.ERROR;
+		if (fullLevel == Level.TRACE) {
 			LOGGER.debug("Upnp log mode: full");
 			SpecificationViolationReporter.enableReporting();
 		} else {
-			if (basicLevel == Level.ALL) {
+			if (basicLevel == Level.TRACE) {
 				LOGGER.debug("Upnp log mode: basic");
-			} else if (mediaServerLevel == Level.ALL) {
+			} else if (mediaServerLevel == Level.TRACE) {
 				LOGGER.debug("Upnp log mode: MediaServer only");
 			} else {
 				LOGGER.debug("Upnp log mode: silent");

